packages/example/pages/_app.tsx

```diff
@@ -85,78 +86,99 @@ const RAINBOW_TERMS = 'https://rainbow.me/terms-of-use';
 const projectId =
   process.env.NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID ?? 'YOUR_PROJECT_ID';
 
-const chains = [
-  mainnet,
-  polygon,
-  optimism,
-  arbitrum,
-  base,
-  zora,
-  bsc,
-  zkSync,
-  ...(process.env.NEXT_PUBLIC_ENABLE_TESTNETS === 'true'
-    ? [
-        goerli,
-        sepolia,
-        holesky,
-        polygonMumbai,
-        optimismSepolia,
-        arbitrumSepolia,
-        baseSepolia,
-        zoraSepolia,
-      ]
-    : []),
-];
-
-const { wallets } = getDefaultWallets({
-  appName: 'RainbowKit Demo',
-  projectId,
-  chains,
-});
+const walletList = getDefaultWalletList();
+
+// If any of the alchemy HTTPS URLs are undefined, wagmi will default
+// to using the first available RPC HTTPS URL from the chain.
+const transports = {
+  [mainnet.id]: http(process.env.NEXT_PUBLIC_RPC_MAINNET_HTTPS),
+  [polygon.id]: http(process.env.NEXT_PUBLIC_RPC_POLYGON_HTTPS),
+  [optimism.id]: http(process.env.NEXT_PUBLIC_RPC_OPTIMISM_HTTPS),
+  [arbitrum.id]: http(process.env.NEXT_PUBLIC_RPC_ARBITRUM_HTTPS),
+  [base.id]: http(process.env.NEXT_PUBLIC_RPC_BASE_HTTPS),
+  [zora.id]: http(process.env.NEXT_PUBLIC_RPC_ZORA_HTTPS),
+  [bsc.id]: http(process.env.NEXT_PUBLIC_RPC_BSC_HTTPS),
+  [zkSync.id]: http(process.env.NEXT_PUBLIC_RPC_ZKSYNC_HTTPS),
+  // Testnet networks
+  [goerli.id]: http(process.env.NEXT_PUBLIC_RPC_GOERLI_HTTPS),
+  [sepolia.id]: http(process.env.NEXT_PUBLIC_RPC_SEPOLIA_HTTPS),
+  [holesky.id]: http(process.env.NEXT_PUBLIC_RPC_HOLESKY_HTTPS),
+  [polygonMumbai.id]: http(process.env.NEXT_PUBLIC_RPC_POLYGON_MUMBAI_HTTPS),
+  [optimismSepolia.id]: http(
+    process.env.NEXT_PUBLIC_RPC_OPTIMISM_SEPOLIA_HTTPS,
+  ),
+  [arbitrumSepolia.id]: http(
+    process.env.NEXT_PUBLIC_RPC_ARBITRUM_SEPOLIA_HTTPS,
+  ),
+  [baseSepolia.id]: http(process.env.NEXT_PUBLIC_RPC_BASE_SEPOLIA_HTTPS),
+  [zoraSepolia.id]: http(process.env.NEXT_PUBLIC_RPC_ZORA_SEPOLIA_HTTPS),
+};
 
-const wagmiConfig = getDefaultConfig({
+const config = getDefaultConfig({
   appName: 'RainbowKit Demo',
   projectId,
-  chains,
-  wallets: [
-    ...wallets,
+  chains: [
+    mainnet,
+    polygon,
+    optimism,
+    arbitrum,
+    base,
+    zora,
+    bsc,
+    zkSync,
+    ...(process.env.NEXT_PUBLIC_ENABLE_TESTNETS === 'true'
+      ? [
+          goerli,
+          sepolia,
+          holesky,
+          polygonMumbai,
+          optimismSepolia,
+          arbitrumSepolia,
+          baseSepolia,
+          zoraSepolia,
+        ]
+      : []),
+  ],
+  transports,
+  walletList: [
```

Comment:
I would just call this `wallets`, since it mirrors the `connectorsForWallets` API

Reply:
The `WalletList` type is generally a bit confusing; I think we'll want to refactor that in the future. `wallets` seems a bit more intuitive as the external prop, even though it is `WalletList` type under the hood